From 378e3037ac09b8851ad83261930295f3eceb590c Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Thu, 29 Sep 2011 14:04:22 +0000
Subject: power: Do not sleep-on-idle by default

We do not actually read the boolean keys in gnome-settings-daemon due to
a mixup when the UI was re-designed. We can just change the default of
the timeout key (that we do read...) to zero to change "never" to be
the default.

Note: we're not actually going to remove the unused keys from the schema
like we did in master, otherwise upgrading the g-s-d package whilst
the control center is open is going to make it explode.

Resolves https://bugzilla.gnome.org/show_bug.cgi?id=660395
---
diff --git a/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in.in b/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in.in
index 665192f..d2d4b48 100644
--- a/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in.in
+++ b/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in.in
@@ -46,7 +46,7 @@
       <description>Whether to put the computer to sleep when inactive on AC power.</description>
     </key>
     <key name="sleep-inactive-ac-timeout" type="i">
-      <default>1800</default>
+      <default>0</default>
       <summary>Sleep timeout computer when on AC</summary>
       <description>The amount of time in seconds the computer on AC power needs to be inactive before it goes to sleep.</description>
     </key>
@@ -61,7 +61,7 @@
       <description>Whether to put the computer to sleep when inactive on battery power.</description>
     </key>
     <key name="sleep-inactive-battery-timeout" type="i">
-      <default>1800</default>
+      <default>0</default>
       <summary>Sleep timeout computer when on battery</summary>
       <description>The amount of time in seconds the computer on battery power needs to be inactive before it goes to sleep.</description>
     </key>
--
cgit v0.9.0.2

